### PR TITLE
refactor(browser): initialize js/browser package

### PR DESCRIPTION
**Pull request by IceHe**
## Summary

- [refactor(browser): initialize js/browser package](https://github.com/logto-io/js/commit/151fdd748a02e8758ea7e323596ca10953d24748)
- [chore(js): remove redundent part of eslintConfig.extends](https://github.com/logto-io/js/pull/157/commits/10a7bda5d59042757bbb3e78c081972787c4ab4a) 

From
```json
  "eslintConfig": {
    "extends": "@silverhand/eslint-config"
```
to
```json
  "eslintConfig": {
    "extends": "@silverhand"
```

---
Deprecated previous #156 due to the wrong branch name.

## Linear Issue Reference

LOG-1466

## Testing

Just add a temp test for passing CI.

---
@logto-io/eng

### CODE REVIEW

File: packages/browser/package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  "scripts": {
+    "dev:tsc": "tsc -b -w --preserveWatchOutput",
+    "preinstall": "npx only-allow pnpm",
+    "precommit": "lint-staged",
+    "build": "tsc",
+    "lint": "eslint --ext .ts src",
+    "package": "webpack",
+    "test": "jest",
+    "test:coverage": "jest --silent --coverage",
+    "test:report": "codecov -F client",
```
Comment (simeng-li) [inline packages/browser/package.json:16]: should have a different report flag.

**Review by demonzoo — APPROVED** [review]

File: packages/browser/package.json (hunk 1 above)
Comment (IceHe) [reply packages/browser/package.json]: Fixed: client → browser. PTAL.

File: packages/browser/package.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    "prettier": "^2.3.2",
+    "ts-jest": "^27.0.4",
+    "ts-loader": "^9.2.6",
+    "typescript": "^4.3.5",
+    "webpack": "^5.68.0",
+    "webpack-bundle-analyzer": "^4.5.0",
+    "webpack-cli": "^4.9.1"
+  },
+  "eslintConfig": {
+    "extends": "@silverhand/eslint-config"
```
Comment (gao-sun) [inline packages/browser/package.json:39]:
```suggestion
    "extends": "@silverhand"
```

**Review by wangsijie — APPROVED** [review]

File: packages/browser/package.json (hunk 2 above)
Comment (IceHe) [reply packages/browser/package.json]: Fixed `js/js` package.json too. PTAL.

**Pull request merged**